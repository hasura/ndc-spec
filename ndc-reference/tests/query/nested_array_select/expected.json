[
  {
    "rows": [
      {
        "id": 1,
        "staff": [
          {
            "last_name": "Landin",
            "fields_of_study": [
              "Computer Science",
              "Education"
            ]
          }
        ],
        "departments": [
          "Humanities and Social Sciences",
          "Science and Engineering",
          "Medicine and Dentistry"
        ]
      },
      {
        "id": 2,
        "staff": [
          {
            "last_name": "Hughes",
            "fields_of_study": [
              "Computer Science",
              "Functional Programming",
              "Software Testing"
            ]
          },
          {
            "last_name": "Claessen",
            "fields_of_study": [
              "Computer Science",
              "Functional Programming",
              "Automated Reasoning"
            ]
          }
        ],
        "departments": [
          "Architecture and Civil Engineering",
          "Computer Science and Engineering",
          "Electrical Engineering",
          "Physics",
          "Industrial and Materials Science"
        ]
      }
    ]
  }
]