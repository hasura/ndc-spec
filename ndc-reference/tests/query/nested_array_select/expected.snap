---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/nested_array_select/request.json
---
[
  {
    "rows": [
      {
        "id": 1,
        "staff": [
          {
            "last_name": "Landin",
            "fields_of_study": [
              "Computer Science",
              "Education"
            ]
          }
        ],
        "departments": [
          "Humanities and Social Sciences",
          "Science and Engineering",
          "Medicine and Dentistry"
        ]
      },
      {
        "id": 2,
        "staff": [
          {
            "last_name": "Hughes",
            "fields_of_study": [
              "Computer Science",
              "Functional Programming",
              "Software Testing"
            ]
          },
          {
            "last_name": "Claessen",
            "fields_of_study": [
              "Computer Science",
              "Functional Programming",
              "Automated Reasoning"
            ]
          }
        ],
        "departments": [
          "Architecture and Civil Engineering",
          "Computer Science and Engineering",
          "Electrical Engineering",
          "Physics",
          "Industrial and Materials Science"
        ]
      },
      {
        "id": 3,
        "staff": [
          {
            "last_name": "Zdravkovic",
            "fields_of_study": [
              "Requirements Engineering",
              "Enterprise Modeling",
              "System Analysis"
            ]
          }
        ],
        "departments": [
          "Computer and Systems Sciences",
          "Physics",
          "Astronomy"
        ]
      }
    ]
  }
]
